internal/receiver/discoveryreceiver/statement_evaluator_test.go

```diff
@@ -105,8 +98,12 @@ func TestStatementEvaluation(t *testing.T) {
 								)
 							}
 
-							wg.Wait()
-							close(plogs)
+							corr := <-emitCh
+							entityEvents, numFailed, err := entityStateEvents(corr.observerID,
+								[]observer.Endpoint{corr.endpoint}, cStore, time.Now())
+							require.NoError(t, err)
+							require.Equal(t, 0, numFailed)
+							emitted := entityEvents.ConvertAndMoveToLogs()
```

Comment:
Do we still want to call this in the test, even though we've removed the call in the statement evaluation itself?

Reply:
As I mentioned in the description, I want to keep the test with minimal changes to showcase the diff. I want to simplify this test as follow up, there is no need to parse the entity event here anymore